### PR TITLE
Add aws shared vpc as a supported workflow

### PR DESCRIPTION
**Pull request by melvinjoseph86**

### CODE REVIEW

**Comment by melvinjoseph86** [comment]
@hoxhaeris @yunjiang29 Kindly review...

**Comment by melvinjoseph86** [comment]
@liangxia Kindly approve

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-byo-route53:
     platform: aws
+  cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts:
+    platform: aws
+  cucushift-installer-rehearse-aws-ipi-shared-vpc-phz:
+    platform: aws
+  cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:47]: Would you mind to sort them in alphanumeric order ?

Comment (melvinjoseph86) [reply core-services/ci-chat-bot/workflows-config.yaml:47]: done

**Pull request merged**